assets/js/apps/search/components/filters/filters-section/FiltersSection.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { Fragment } from 'react';
+import Tabs from "react-bootstrap/Tabs";
+import Tab from 'react-bootstrap/Tab';
+import { OBJECT_TYPE_STICKERS } from '../../TabStickers/TabSticker'
+import TypeSchemaList from '../type-schema-list/TypeSchemaList';
+import { runSearch } from '../../searchSlice';
+import { typeAttrSelector, allTypeAttrIdsSelector, updateActiveSchemas, updateTypeAttribute } from '../filterSlice';
+import { useSelector, useDispatch, batch } from "react-redux";
+import PropTypes from "prop-types";
+import { mapTypeToFilter } from "../index";
+
+
+function TypeAttributeFilter({typeId, attributeId}) {
+  const attribute = useSelector(state => (typeAttrSelector(state.filters,typeId,attributeId)));
+  const dispatch = useDispatch();
+  const setFilterValue = value => {
+    batch(() => {
+      dispatch(updateTypeAttribute({
+        typeId,
+        attributeId,
+        value
+      }));
+      dispatch(runSearch());
+    });
+  };
+  const ApplicableFilter = mapTypeToFilter(attribute.data_type);
+  return (
+    <section>
+      <h3 className="h5">{attribute.full_name}</h3>
+      <ApplicableFilter id={typeId+"."+attributeId} value={attribute.value} onValueChange={setFilterValue} options={attribute.options} />
```

Comment:
Codacy has a fix for the issue: [Operator '+' must be spaced.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933948)
```suggestion
      <ApplicableFilter id={typeId + "."+attributeId} value={attribute.value} onValueChange={setFilterValue} options={attribute.options} />
```